torch/onnx/symbolic_opset9.py

```diff
@@ -2460,75 +2526,117 @@ def convolution(
 
 
 @_onnx_symbolic("aten::conv1d")
-@symbolic_helper.parse_args("v", "v", "v", "is", "is", "is", "i")
+@symbolic_helper.parse_args("v", "v", "v", "is", "v", "is", "i")
 @_beartype.beartype
 def conv1d(
     g: jit_utils.GraphContext, input, weight, bias, stride, padding, dilation, groups
 ):
-    return _convolution(
-        g,
-        input,
-        weight,
-        bias,
-        stride,
-        padding,
-        dilation,
-        False,
-        (),
-        groups,
-        None,
-        None,
-        None,
-        None,
-    )
+    str_padding = symbolic_helper._parse_arg(padding, "s")
```

Comment:
Wondering if we need to parse_arg here, as we already have one on `convolution_mode` decorator?